README.md

```diff
@@ -34,7 +34,7 @@ Why not use thunks? Thunks are simple and might be better suited to your applica
 
 * More control over your side effects. For instance you can use take-latest to cancel an existing take operation in progress. You can use take to pause until a action is fired. These would be difficult to achieve with thunks.
 * It can keep your actions simple and boilerplate and move side-effects to a seperate file.
-* The middleware is just triggered by actions, meaning if you have a store multi-window, you can run the side-effects in a different window as it only relies on listening to pure actions.
+* The middleware is just triggered by actions, meaning if you have a store in multi-windows, you can run the side-effects in a different window as it only relies on listening to pure actions.
```

Comment:
Not much better!

Reply:
Ha, true. It read OK in my head...

Is `The middleware is just triggered by actions, meaning if you have a single store across multiple windows,` better?